Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstanding.cs

```diff
@@ -256,6 +265,7 @@ private void Update_Scan(float deltaTime)
 
                 // Mark it
                 ScanState = ScanStates.Done;
+                OnScanDone.Invoke();
```

Comment:
No worries, essentially you're just checking to make sure there's subscribers before invoking the event.